gadget/gadget.go

```diff
@@ -47,11 +47,13 @@ const (
 
 	SystemBoot = "system-boot"
 	SystemData = "system-data"
+	SystemSeed = "system-seed"
 	BootImage  = "bootimg"
 	BootSelect = "bootselect"
 	// ImplicitSystemDataLabel is the implicit filesystem label of structure
 	// of system-data role
 	ImplicitSystemDataLabel = "writable"
+	SystemDataLabel         = "ubuntu-data"
```

Comment:
I'm a bit confused by this, so writable is passed right now either explicitly in the structure, or implicitly based on role being `system-data`. Does this mean that when `system-seed` role is present the `system-data` gets a different label now?

Reply:
Yes. When `system-seed` is present, the system should use London names, i.e. `system-data` label should be `ubuntu-data` and not `writable` anymore. Labels are different in Core 20.

Reply:
Should this be named `UbuntuSystemDataLabel` then?

Reply:
Renamed.